core/build.gradle.kts

```diff
@@ -74,16 +74,15 @@ tasks.test {
   finalizedBy("dockerStop", "dockerRemoveContainer")
 }
 
-tasks.dokka {
-  outputFormat = "html"
-  outputDirectory = "$buildDir/javadoc"
+tasks.dokkaHtml {
```

Comment:
do we really need dokka?

Reply:
There are no special requirements. Javadoc may be fake.
https://central.sonatype.org/publish/requirements/ 
>If, for some reason (for example, license issue or it's a Scala project), you can not provide -sources.jar or -javadoc.jar , please make fake -sources.jar or -javadoc.jar with simple README inside to pass the checking. We do not want to disable the rules because some people tend to skip it if they have an option and we want to keep the quality of the user experience as high as possible.